### PR TITLE
provide a list of speaker instead of just one

### PR DESCRIPTION
**Pull request by gorghoa**
I understand that I create a possible impossible state with my List of speakers (the case where the list is empty). If anyone has a hint how to solve this, I’m all ears.

### CODE REVIEW

File: Card.elm [hunk 1]
```diff
@@ -14,17 +14,22 @@ import String
...
-        [ displaySpeaker talk.speaker
+        [ displaySpeakers talk.speakers
         , displayTalkInfo talk
         ]
 
 
-displaySpeaker : Speaker -> Html a
-displaySpeaker { fullName, profileUrl } =
+displaySpeakers : List Speaker -> Html a
+displaySpeakers speakers =
```
Comment (3rwww1) [inline Card.elm:23]: You can have a `Speakers` type that is in fact a tuple holding both the mandatory main `Speaker` and a list of additional `Speakers` that we'll call `featurings`

``` elm
displaySpeakers: Speakers -> Html a
displaySpeakers (main, featurings) =
```

File: Card.elm [hunk 2]
```diff
@@ -14,17 +14,22 @@ import String
...
     let
         layout =
             styles [ flex (pc 25) ]
+    in
+        div [ layout ] (List.map displaySpeaker speakers)
```
Comment (3rwww1) [inline Card.elm:28]: The `::` (cons) operator makes the update almost seemless here:

``` elm
        div [ layout ] (List.map displaySpeaker (main :: featurings))
```

File: Meetups.elm [hunk 3]
```diff
@@ -4,7 +4,7 @@ import Speakers exposing (..)
```
Comment (3rwww1) [inline Meetups.elm:4]: in Speakers.elm, you could add a `type alias Speakers` which would translate to a tuple: 

``` elm
type alias Speakers = (Speaker, List Speaker)
```

Or you could also use a record if you're picky about naming:

``` elm
type alias Speakers =
  { main : Speaker
  , featurings : List Speaker
  }
```

This is a tad more verbose (matter of taste I guess) and not compatible with my other code comments

File: Meetups.elm [hunk 4]
```diff
@@ -4,7 +4,7 @@ import Speakers exposing (..)
 
 
 type alias Talk =
-    { speaker : Speaker
+    { speakers : List Speaker
```
Comment (3rwww1) [inline Meetups.elm:7]:
``` elm
    { speakers : Speakers
```

File: Meetups.elm [hunk 5]
```diff
@@ -34,19 +34,19 @@ meetup1 =
     { date = "2016-03-24"
     , location = "Meetic"
     , lineUp =
-        [ { speaker = tbrisbout
+        [ { speakers = [ tbrisbout ]
```
Comment (3rwww1) [inline Meetups.elm:37]: Single speaker talks would need to be changed as in:

``` elm
        [ { speakers = (tbrisbout, [])
```

Whereas multiple-speakers talks would go like this:

``` elm
        [ { speakers = (sebcreme, [ erwan ])
```

File: Meetups.elm [hunk 6]
```diff
@@ -88,25 +88,25 @@ meetup3 =
...
           }
-        , { speaker = klebrun
+        , { speakers = [ klebrun ]
           , title = "Migration d'une application vers Elm 0.17"
           , description = "Ce talk fera le point sur les nouveautés de la version 0.17 via un retour d'expérience pratique sur la mise à jour d'une application de elm 0.16 à 0.17, comment migrer les signaux vers les subscriptions / commandes / messages, en décrivant les nouveaux patterns à utiliser."
           , slidesUrl = Just "http://slides.com/kevinlebrun/from-elm016-to-elm017/fullscreen"
           , videoUrl = Just "https://youtu.be/c9E9_vSaeDo?list=PLFmYNJ_DUU2DPZH2Vfok0jdfkGPcrL0JA"
           }
-        , { speaker = abaumann
+        , { speakers = [ abaumann ]
```
Comment (3rwww1) [inline Meetups.elm:109]: As far as I can remember, Alick Paul always talked along `abaumann`

**Review by 3rwww1 — COMMENTED** [review]
Ensuring a single speaker is possible (see my comments), albeit a bit awkward and maybe unnecessary - but I guess that's up to @tbrisbout!

**Comment by tbrisbout** [comment]
I will merge it like that since the feature is working. The impossible state concern is a little overkill IMO since we have control on the original data directly in Elm (for now). Thanks for the work and your ideas !

**Pull request merged**